webapp/components/needs_team.jsx

```diff
@@ -42,20 +42,17 @@ export default class NeedsTeam extends React.Component {
         this.onChanged = this.onChanged.bind(this);
 
         this.state = {
-            profiles: UserStore.getProfiles(),
```

Comment:
What happens if Profiles change after TeamsChange?  then the component will not re-render?

I switch teams.  fires getMyTeam and getProfiles(forNewTeamId) and getMyTeam finishes before getProfiles, won't it be showing the old profiles?